src/lib/orm/creator.go

```diff
@@ -15,7 +15,6 @@
 package orm
 
 import "github.com/beego/beego/orm"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380